app/assets/stylesheets/src/ats/_candidate.scss

```diff
@@ -13,6 +9,11 @@
       }
     }
 
+    .card-footer {
+      border-top-color: var(--tblr-border-color);
+      padding: 16px 20px;
```

Comment:
in general we need to set padding left and right, but I also set top and bottom to look similar as padding definition for `.compose-form-message`